docs/xbox.md

```diff
@@ -0,0 +1,29 @@
+---
+section: legacy
+hide:
+  - next
+  - prev
+  - search
+  - tabs
+---
+
+## Input Plugin
+
+The recommended controller plugin for XBox and other USB controllers is N-Rage V2 1.83, which is included with the emulator. Make sure to select it as your input plugin by going to `Options -> Settings -> Plugins.` These instructions **won't work** if you're using a different input plugin.
```

Comment:
I think we should have an explicit mention that this doesn't work for 2.23. Even as simple as:
> These instructions **won't work** if you're using a different input plugin, like `N-Rage Input Plugin V2 2.3c`.

Reply:
Added in https://github.com/smash64-dev/smash64-online/pull/10/commits/f93c24ee6dd89005818eb97d9a97a58d4f2a76e2